pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -31,9 +31,63 @@ local zotero = require('zotero')
 -- -- global state -- --
 local config = {
   client = 'zotero',
-  scannable_cite = false
+  scannable_cite = false,
+  transferable = false,
+  csl_style='',
+  dont_fill_plaincitation = false
 }
 
+-- -- -- bibliography marker generator -- -- --
+local function zotero_bibl()
+
+  if config.scannable_cite then
+    return ''
+  end
+
+  local mycsl = '{&quot;uncited&quot;:[],&quot;omitted&quot;:[],&quot;custom&quot;:[]}'
```

Comment:
> local mycsl = '{&quot;uncited&quot;:[],&quot;omitted&quot;:[],&quot;custom&quot;:[]}'

I _assume_ it tells how the bibliography is generated.  For example, should the bibliography include works that are _not_ in the main document ... etc etc.  

Let us call it `bibpreferences`.

local something=<... a lua value...>
local bibpreferences=json.encode(something)

Tell me what the lua value expression for something should be ... _I tried something it gave me `{}` in json expression, instead of the required `[]`.  I need your help here.